src/components/ContactOption.jsx

```diff
@@ -10,12 +10,15 @@ function ContactOption(props) {
   const { onSelectContact } = props;
   const [contacts, setContacts] = useState([]);
 
-  const [defaultValue, setDefaultValue] = useState(contacts[0].name);
+  const [defaultValue, setDefaultValue] = useState(null);
 
   const fetchContacts = () => {
     fetch(BASE_URL, { method: 'GET' })
       .then((response) => response.json())
-      .then((data) => setContacts(data));
+      .then((data) => {
+        setContacts(data);
+        setDefaultValue(data[0].id);
```

Comment:
Assumes that `data` is non-empty

Reply:
I'll update it to make it work for empty data

Reply:
Just updated it now